packages/paste-website/src/pages/patterns/filter-group/index.mdx

```diff
@@ -194,11 +179,7 @@ While we don’t yet have an official custom date/time component, in the interim
 
 When the Custom option is selected, the Popover containing the Date Inputs and Time Inputs necessary would be displayed to allow customers to enter their desired range. Once the Apply button is clicked, the Popover would hide. If a customer wants to change their original selections, they must click on the Calendar link style button to open the Popover again.
```

Comment:
question: the popover doesn't open for me when the custom option is selected, only when the calendar icon is clicked. this makes it a kind of confusing interaction because I can select custom without knowing how to actually specify the custom dates. similarly, if I click the cal icon to open the date range popover, and click "apply", it doesn't switch the menu selection back to "custom" so it looks like the previous selection is still being applied. lmk if this is confusing and I can explain on a zoom/huddle

Reply:
just saw your comment about how shifting focus on click of "custom" wouldn't be accessible... I wonder if we can add some text to clarify the interaction? like "Custom (click cal icon to specify dates)" or something less wordy but to the same point. maybe just me but I was definitely confused when using this filter